tests/js/spec/components/group/sentryAppExternalIssueActions.spec.jsx

```diff
@@ -138,7 +139,6 @@ describe('SentryAppExternalIssueActions', () => {
 
       wrapper.find('Input#title').simulate('change', {target: {value: 'foo'}});
       wrapper.find('TextArea#description').simulate('change', {target: {value: 'bar'}});
-      selectByValue(wrapper, 1, {name: 'numbers'});
```

Comment:
Do we not want to test selecting using the numbers field?

Reply:
@billyvg We do in `sentryAppExternalIssueForm.spec.jsx`, for whatever reason I couldn't get setting the value from the select working in this spec file. I didn't want to spend more time on it since we do test it in that other file.